Languages/Android/questions.json

```diff
@@ -577,6 +577,33 @@
       "trueOrFalse": false,
       "shortAns": "It allows foreign applications to execute our app's code",
       "tag": "pending_intent"
+    },
+    {
+      "id": 64,
+      "question": "What is paging in android",
+      "details": "Paging is an Android Jetpack feature that is used to load and display small amounts of data from a remote server. It uses less network bandwidth. ",
+      "questionType": "multi ",
+      "trueOrFalse": false,
+      "shortAns": "It is used to load in the form of small chunks.",
```

Comment:
May you rephrase your short answer from ("small chunks." to "small amounts of data.")? @glitched-sudhanshu 

Thank you for participation!